src/guest_memory.rs

```diff
@@ -288,6 +288,11 @@ pub trait GuestMemoryRegion: Bytes<MemoryRegionAddress, E = Error> {
     fn as_volatile_slice(&self) -> Result<volatile_memory::VolatileSlice> {
         self.get_slice(MemoryRegionAddress(0), self.len() as usize)
     }
+
+    /// Returns true if the region is hugepages
+    fn is_hugepages(&self) -> bool {
```

Comment:
Does hugepage here mean Hugetlbfs? Or  it also includes Transparent Huge Pages?

Reply:
Yes, hugepage means Hugetlbfs.
THPs can be release as normal pages.

Reply:
Hugepage is a little ambiguous, should we clearly state that it's hugetlbfs?